README.md

```diff
@@ -5,15 +5,15 @@
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=sonarscanner-msbuild&metric=coverage)](https://sonarcloud.io/component_measures?id=sonarscanner-msbuild&metric=coverage)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](#contributing)
 
-The SonarScanner for .NET is the recommended way to launch a SonarQube or SonarCloud analysis for projects/solutions using MSBuild or dotnet command as build tool. It is the result of a [collaboration between SonarSource and Microsoft.](http://www.sonarqube.org/announcing-sonarqube-integration-with-msbuild-and-team-build/)
+The SonarScanner for .NET is the recommended way to launch a SonarQube or SonarCloud analysis for [Clean Code](https://www.sonarsource.com/solutions/clean-code/) projects/solutions using MSBuild or dotnet command as a build tool. It is the result of a [collaboration between SonarSource and Microsoft.](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-dotnet/)
```

Comment:
The `collaboration between SonarSrouce and Microsoft` now leads to `SonarScanner for .NET` documentation page, that again has a `collaboration between SonarSource and Microsoft` link pointing to the final destination.

That behavior is unexpected from a reader point of view. Let's directly go to the announced target
```suggestion
The SonarScanner for .NET is the recommended way to launch a SonarQube or SonarCloud analysis for [Clean Code](https://www.sonarsource.com/solutions/clean-code/) projects/solutions using MSBuild or dotnet command as a build tool. It is the result of a [collaboration between SonarSource and Microsoft.](https://www.sonarsource.com/blog/easy-analysis-of-visual-studio-solutions-with-the-sonarqube-scanner-for-msbuild/)
```